### PR TITLE
FEAT: added homepage.

### PR DESCRIPTION
**Pull request by zoyatanweer**
In this PR I have added : 

- homepage 
- header
- homepage cover
- added svg
- featured categories
- footer

Things not to check:
- CSS files
- .json file

### CODE REVIEW

File: src/Assets/Svg/allsvg.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      <path
+        fill="#000000"
+        d="M15.5 14h-.79l-.28-.27A6.471 6.471 0 0 0 16 9.5A6.5 6.5 0 1 0 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5S14 7.01 14 9.5S11.99 14 9.5 14z"
+      ></path>
+    </svg>
+  );
+}
+function ShoppingCartIcon(props) {
+  return (
+    // <svg width="1em" height="1em" viewBox="0 0 24 24" {...props}>
```
Comment (Muzammil-khan710) [inline src/Assets/Svg/allsvg.jsx:13]: Remove this if you won't use in future

**Review by Muzammil-khan710 — APPROVED** [review]

**Comment by Muzammil-khan710** [comment]
Nice UI
Clean code !
Good to merge !

File: src/components/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export { NavLinkItems } from "./header/NavLinkItems";
+export { NavLinkUser } from "./header/NavLinkUser";
```
Comment (srishtigautam21) [inline src/components/index.js:2]: i think import should be here in place of export
it should be import { NavLinkItems } from "./header/NavLinkItems";
then export {NavLinkItems}

**Review by srishtigautam21 — APPROVED** [review]
UI looks good
Code is also good to go with the suggested changes

File: src/Assets/Svg/allsvg.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    //   <path
+    //     fill="#888888"
+    //     d="M15.55 13c.75 0 1.41-.41 1.75-1.03l3.58-6.49A.996.996 0 0 0 20.01 4H5.21l-.94-2H1v2h2l3.6 7.59l-1.35 2.44C4.52 15.37 5.48 17 7 17h12v-2H7l1.1-2h7.45zM6.16 6h12.15l-2.76 5H8.53L6.16 6zM7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2s-.9-2-2-2zm10 0c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2s2-.9 2-2s-.9-2-2-2z"
+    //   ></path>
+    // </svg>
```
Comment (zoef-yt) [inline src/Assets/Svg/allsvg.jsx:18]: Remove commented code.

File: src/Assets/Images/search_black_24dp.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"/></svg>
```
Comment (zoef-yt) [inline src/Assets/Images/search_black_24dp.svg:1]: If you have one `allsvg.jsx ` where you have all the react icons why have this?

File: src/components/header/NavLinkUser.jsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+export const NavLinkUser = (props) => {
+  return (
+    <div className="single-action-btn profile">
+      <a href="">{props.svg}</a>
```
Comment (zoef-yt) [inline src/components/header/NavLinkUser.jsx:5]: As you are just using 1 prop destructure it at the top only

**Review by zoef-yt — APPROVED** [review]
The code looks good. The UI looks amazing. Good work zoya. good to merge with changes.

File: src/Assets/Images/search_black_24dp.svg (hunk 4 above)
Comment (zoyatanweer) [reply src/Assets/Images/search_black_24dp.svg:1]: thanks for pointing out, will remove it

**Comment by zoyatanweer** [comment]
Thankyou for the reviews guys.

**Comment by zoyatanweer** [comment]
Thankyou for the reviews guys.

**Pull request merged**